### PR TITLE
Spark version/2.0

### PR DESCRIPTION
**Pull request by andypetrella**
There we are...

cc @vidma

### CODE REVIEW

File: modules/spark/src/main/scala_2.11/spark-last/Boot.scala [hunk 1]
```diff
@@ -6,12 +6,12 @@ import scala.tools.nsc.Settings
...
 
 object Boot {
-  def classServer(outputDir:File) = {
+  /*def classServer(outputDir:File) = {
     val conf = new SparkConf()
     val server = new HttpServer(conf, outputDir, new SecurityManager(conf))
     server
-  }
-  
+  }*/
```
Comment (vidma) [inline modules/spark/src/main/scala_2.11/spark-last/Boot.scala]: why do you comment this out?

maybe delete or at least add a `FIXME` comment, if it needs attention in the future.

File: modules/common/src/main/ds/DownloadCsv.scala [hunk 2]
```diff
@@ -0,0 +1,77 @@
+package notebook.front.widgets
```
Comment (vidma) [inline modules/common/src/main/ds/DownloadCsv.scala]: let's remove `df/DownloadCsv.scala`, and use only native spark csv. no more issues with lib versioning.

**Comment by andypetrella** [comment]
the future-master branch takes care of that

File: modules/common/src/main/ds/DownloadCsv.scala (hunk 2 above)
Comment (andypetrella) [inline modules/common/src/main/ds/DownloadCsv.scala]: I'm okay with that, could you proceed maybe?
don't want to make mistakes ^^

**Pull request closed without merging**